src/main/resources/assets/techreborn/lang/fr_fr.lang

```diff
@@ -898,3 +898,6 @@ achievement.TechReborn:ore_Pickup=Nouveau minerai !
 achievement.TechReborn:ore_Pickup.desc=Minez un minerai techreborn
 achievement.TechReborn:centrifuge_Craft=Il est temps de tourner
 achievement.TechReborn:thermalgen_Craft=Il est temps de chauffer
+
+#Blocks
+tile.techreborn:alarm.name=Alarme
```

Comment:
Im not sure how you go theses, just checking if you did use google translate or something done.

Reply:
I literally typed alarm to each language and confirmed with other sources besides  Google translate, they should prove a valid placeholder till someone can confirm.

Reply:
Such a simple word 'Alarm', it's pretty easy to find the proper name in other languages. As alarm is generally a universally used word.